src/index.js

```diff
@@ -38,15 +39,16 @@ import CarInsuranceIcon from './icons/carInsurance.svg';
 import CashIcon from './icons/cash.svg';
 import CashPaymentBillIcon from './icons/cashPaymentBill.svg';
 import CashPaymentIcon from './icons/cashPayment.svg';
-import Chat from './icons/chat.svg';
-import CheckboxChecked from './icons/checkboxChecked.svg';
-import CheckboxUnchecked from './icons/checkboxUnchecked.svg';
+import ChatIcon from './icons/chat.svg';
+import CheckboxCheckedIcon from './icons/checkboxChecked.svg';
```

Comment:
Thanks for renaming these, we will need to be careful to remember to update the imports (and markup) used for this icon in the UI Library (same for the next line, and same for any other older icons that are being renamed in this PR)

Reply:
Yes, exactly!! We maybe should make it a major version!!